### PR TITLE
Initialized Cell and test

### PR DESCRIPTION
**Pull request by GDemps**
I just wanted to get the feel for the process, so check it out and make sure I spelled initialized correctly.  :)

### CODE REVIEW

File: test/Cell_test.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/Cell'
+require './lib/Ship'
+
+require "pry"
```
Comment (muydanny) [inline test/Cell_test.rb]: Great Grant! We can take this pry out  and binding.pry and we'll be good! Great way for me to learn how to comment lol

**Review by muydanny — CHANGES_REQUESTED** [review]

File: lib/Cell.rb [hunk 2]
```diff
@@ -2,8 +2,9 @@ class Cell
 require './lib/Ship'
 require 'pry'
```
Comment (muydanny) [inline lib/Cell.rb]: Found another pry lol

**Review by muydanny — CHANGES_REQUESTED** [review]

File: lib/Cell.rb (hunk 2 above)
Comment (GDemps) [reply lib/Cell.rb]: cool, I'll take it out

**Review by muydanny — APPROVED** [review]
Looks great partner!

**Pull request merged**